### DISCUSSION TITLE
Download documents every month from the Crowd strike portal - API script for automate the task

### DISCUSSION THREAD

**Discussion by afroz1** [CrowdStrike/falconpy#822] [Q&A] [answered]
Hi,

I am looking for a python script that we can use to download updated Crowdstrike documents to an isolated network.  Kindly help or Guide me if you can support me in making some custom script for it.

**Comment by crowdstrikedcs** [comment, marked as answer]
Hi @afroz1 

Thanks for your question! Converting this issue to a discussion here. 

Are you able to elaborate a bit more on your use case? Let me know a bit more detail about these "CrowdStrike documents" if you could please. 

We do have the ability to download the CrowdStrike sensor over API, this is the agent installed on endpoints. I'd also like to learn a bit more about your "isolated network" as using the CrowdStrike Agent with an internet connection allows for more functionality.

**Reply by afroz1** [reply]
Hi,

Every 2-3 months,  I have to download some updated documents from the Crowdstrike EU-1 portal to our share drive which I need to automate.  Need to call crowd strike API (where all the sensor deployment documents are available) and upload them in our share drive that would be automate no manual effort required.

Just an Example for the document:

1. Cloud IP Addresses _ Documentation _ Support and resources _ Falcon
2. Container Security _ Documentation _ Support and resources _ Falcon
3. Falcon Sensor for Windows Deployment _ Documentation _ Support and resources _ Falcon
4. Falcon Sensor for Linux Deployment _ Documentation
5. Falcon Sensor for Mac Deployment _ Documentation _ Support and resources _ Falcon
6. Supported Operating Systems
7. etc

**Reply by crowdstrikedcs** [reply]
Got it, currently we aren't able to access this info over API. So this would require our product team making the required changes. I was able to find the below ideas where others have requested this capability. I would encourage you to vote and comment on these so that the product team may prioritize them appropriately. 

https://eu-1.ideas.crowdstrike.com/ideas/IDEA-I-7262
https://eu-1.ideas.crowdstrike.com/ideas/IDEA-I-8578
